extensions/amp-ad-exit/0.1/amp-ad-exit.js

```diff
@@ -251,9 +253,14 @@ export class AmpAdExit extends AMP.BaseElement {
         'be inside a <script> tag with type="application/json"');
     try {
       const config = assertConfig(parseJson(child.textContent));
+      if (isObject(config.options) &&
+          typeof config.options.startTimingEvent === 'string') {
+        this.defaultFilters_.splice(0, 1, createFilter('minDelay',
```

Comment:
IMO options.startTimingEvent should also apply to user filters unless a different event is specified. What about passing options into createFilter and/or individual filter ctors:
https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad-exit/0.1/filters/factory.js#L27